data/human/fleets.txt

```diff
@@ -1637,6 +1637,103 @@ fleet "Large Syndicate"
 		"Vanguard (Missile)"
 		"Quicksilver (Proton)" 2
 
+fleet "Small Syndicate Core"
+	government "Syndicate"
+	names "syndicate small"
+	cargo 2
+	personality
+		appeasing disables plunders
```

Comment:
I believe that removing `disables` but keeping `plunder` results in the behavior "disables to plunder, then destroys after plundering," while having `disables` here is making it so that these fleets try to leave ships disabled. That to me seems like much more sensible behavior; these Syndicate fleets would be plundering what they can, then destroying the ship so no one else can get their hands on it. And in fact, it might make sense to just add `plunders` to the existing Syndicate fleets instead of adding new ones just for the Korath.

Reply:
Agreed. Plunder then destroy fits the Syndicate mentality much better than simply "disable and leave them".